src/main/java/jp/co/future/uroborosql/event/subscriber/DumpResultEventSubscriber.java

```diff
@@ -36,34 +35,38 @@
  * @author H.Sugimoto
  *
  */
-public class DumpResultSqlFilter extends AbstractSqlFilter {
+public class DumpResultEventSubscriber extends EventSubscriber {
 	/** ロガー */
-	private static final Logger LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.filter");
+	private static final Logger EVENT_LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.event");
 
 	/** 文字数計算用のエンコーディング */
 	private static final String ENCODING_SHIFT_JIS = "Shift-JIS";
 
 	/**
+	 *
 	 * {@inheritDoc}
 	 *
-	 * @see jp.co.future.uroborosql.filter.AbstractSqlFilter#doQuery(jp.co.future.uroborosql.context.ExecutionContext, java.sql.PreparedStatement, java.sql.ResultSet)
+	 * @see jp.co.future.uroborosql.event.subscriber.EventSubscriber#initialize()
 	 */
 	@Override
-	public ResultSet doQuery(final ExecutionContext executionContext, final PreparedStatement preparedStatement,
-			final ResultSet resultSet) {
+	public void initialize() {
+		sqlQueryListener(this::sqlQuery);
+	}
+
+	void sqlQuery(final SqlQueryEvent evt) {
 		try {
-			if (resultSet.getType() == ResultSet.TYPE_FORWARD_ONLY) {
-				LOG.warn(
-						"ResultSet type is TYPE_FORWARD_ONLY. DumpResultSqlFilter use ResultSet#beforeFirst(). Please Set TYPE_SCROLL_INSENSITIVE or TYPE_SCROLL_SENSITIVE.");
+			if (evt.getResultSet().getType() == ResultSet.TYPE_FORWARD_ONLY) {
+				if (EVENT_LOG.isWarnEnabled()) {
+					EVENT_LOG.warn(
+							"ResultSet type is TYPE_FORWARD_ONLY. DumpResultEventSubscriber use ResultSet#beforeFirst(). Please Set TYPE_SCROLL_INSENSITIVE or TYPE_SCROLL_SENSITIVE.");
+				}
 			}
-			if (LOG.isInfoEnabled()) {
-				var builder = displayResult(resultSet);
-				LOG.debug("{}", builder);
+			if (EVENT_LOG.isDebugEnabled()) {
+				EVENT_LOG.debug("{}", displayResult(evt.getResultSet()));
 			}
 		} catch (SQLException e) {
 			e.printStackTrace();
```

Comment:
Take this opportunity to modify printStackTrace to the logger.

Reply:
Grep the process of calling printStackTrace and correct it to output to Logger.

Reply:
In addition, the variable names of exceptions have been unified to `ex`, since they had been shaking between `e` and `ex`.